pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2833,6 +2833,9 @@ type StorageScopeSpec struct {
 	// +nullable
 	// +optional
 	StorageClassDeviceSets []StorageClassDeviceSet `json:"storageClassDeviceSets,omitempty"`
+	// resize the osd crush weight once size of the osd pvc is increased
+	// +optional
+	ResizeOsdCrushWeight bool `json:"resizeOsdCrushWeight,omitempty"`
```

Comment:
How about moving this down next to the `AllowDeviceClassUpdate` setting? Also, suggest this name for consistency.
```suggestion
	AllowOsdWeightUpdate bool `json:"resizeOsdCrushWeight,omitempty"`
```

Reply:
AllowOsdCrushWeightUpdate